common/djangoapps/entitlements/views.py

```diff
@@ -0,0 +1,46 @@
+import uuid
```

Comment:
Pretty much everything in this PR belongs in `common/djangoapps/entitlements/api/v1/` on disk. This way, if you have a v2, you need only make a new module. See the layout of https://github.com/edx/edx-platform/tree/master/cms/djangoapps/api/v1.